File: 9-palindrome-number/9-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0){
            return false;
        }else{
            string origin = to_string(x);
            reverse(origin.begin(),origin.end());
            if(origin == to_string(x)){
                return true;
            }else{
                return false;
            }
        }
    }
};

File: 9-palindrome-number/9-palindrome-number.go
func Reverse(s string) (result string) {
  for _,v := range s {
    result = string(v) + result
  }
  return 
}

func isPalindrome(x int) bool {
    if(x<0){
        return false
    }
    str := strconv.Itoa(x)
    if(str==Reverse(str)){
        return true
    }
    return false
}